test/groovy/DockerExecuteOnKubernetesTest.groovy

```diff
@@ -1,12 +1,20 @@
-import com.sap.piper.JenkinsUtils
+import static org.hamcrest.Matchers.*
+
+import static org.hamcrest.Matchers.*
```

Comment:
Two times the same import. I would like to suggest to use no bulk imports, since it is more confusing compared to indivudal imports. Especially when having a look at the code outside an IDE, e.g. just in the github web ui.